{"name": "测试用例描述：错误的账号登录失败", "status": "passed", "description": "测试描述：错误的账号登录失败", "steps": [{"name": "输入错误账号", "status": "passed", "attachments": [{"name": "错误账号", "source": "9c1cc0ac-c3af-416e-95e3-817ce1e0e415-attachment.attach"}], "start": 1651817993729, "stop": 1651817993730}, {"name": "输入正确密码", "status": "passed", "attachments": [{"name": "正确密码", "source": "754ba484-2946-4534-b2fc-0a8df40083db-attachment.attach"}], "start": 1651817993796, "stop": 1651817993797}, {"name": "输入正确验证码", "status": "passed", "attachments": [{"name": "正确验证码", "source": "6a9a1f4d-865b-4068-9a0f-b31132c5bb8b-attachment.attach"}], "start": 1651817993846, "stop": 1651817993847}], "attachments": [{"name": "截图", "source": "8c5da2c6-3c4d-4b4c-8f59-ec9e49908b2c-attachment.png", "type": "image/png"}], "start": 1651817993728, "stop": 1651817996188, "uuid": "3af9bf39-58c2-467a-92d0-d687f569bae9", "historyId": "24e424fe5694bafaaba56099b659de37", "testCaseId": "938631cc2602a20c3281e7f98df5b2bf", "fullName": "test_two.Test#test_05", "labels": [{"name": "epic", "value": "xxx测试"}, {"name": "story", "value": "登录"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "2072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}